src/components/Equipments.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {Image, View, Text, StyleSheet} from 'react-native';
+import sword from '../assets/images/icon-sword.png';
+import shield from '../assets/images/icon-shield.png';
+import agility from '../assets/images/icon-agility.png';
+
+export const Equipments = ({item}) => {
+  const image = () => {
```

Comment:
You could create a separate component that receives the affected_attribute as a param and return the correct image. And use it like: `<EffectImage atb={item.affected_atribute} />`